docs/source/linchpincli_linchpin_layout.rst

```diff
@@ -1,26 +0,0 @@
-Layout
```

Comment:
Can we just comment this out for now? This functionality will return in the near future.